tbx/project_styleguide/templates/patterns/atoms/icons/quote-icon.html

```diff
@@ -5,8 +5,8 @@
         </mask>
         <g mask="url(#{{ mask_id }})" transform="translate(-25.84 3.44)">
             <path fill="currentColor" d="M-11-312.4h1351v696H-11z" class="quote-icon__white" />
-            <path d="M8.59 6.934c.1 1.181 1.73 1.554 2.406.558 3.973-5.98 9.391-11.01 13.069-12.448l-.032.66c-1.405 42.635 30.37 67.332 41.634 74.642 1.065.713 2.448-.322 1.953-1.449-6.89-15.108-3.408-28.904 9.999-41.85 2.545-2.375 5.36-5.472 7.785-8.588 0 0 2.918-3.607 5.746-8.615l.15.155c4.888 7.734 5.635 16.737 5.636 21.662.022 1.177 1.566 1.693 2.313.847 3.491-4.093 9.539-12.472 10.852-21.89 3.126-22.78-10.24-46.876-34.836-54.319-1.16-.35-2.077.781-1.418 1.769 2.038 3.333 3.152 7.944 2.872 13.81C76.072-14.556 59.433 5.29 59.433 5.29S44.76-16.077 45.407-29.643c.283-5.94 1.83-10.422 4.177-13.542a1.205 1.205 0 00-1.244-1.898C23.872-40.392 11.524-20.045 9.19-7.296c-1.052 5.609-.83 10.838-.6 14.23z" fill="var(--color--accent-two)" />
```

Comment:
Query: are the various changes to icons here relevant to this MR?

Reply:
I forgot to mention but I think the MR is out of date as i've seen these changes before from Albina. These are the commits I reviewed as part of this one:

- https://github.com/torchbox/torchbox.com/pull/334/commits/10b5b4d7d5667c80488276ec65bd5aeb2c312d24
- https://github.com/torchbox/torchbox.com/pull/334/commits/35bd38e2001c30339d4ce282b30bd77276715240
- https://github.com/torchbox/torchbox.com/pull/334/commits/f412fb45ac955fff71377af36a45c9ec2580454b

Reply:
Hi @helenb, I initially picked the wrong branch to merge into for this PR and later switched it. I’ve already rebased the branch and updated it already from feature/twe-10-stats-block, so everything is up to date on my end. It looks like some changes are still showing up in the diff, probably due to a GitHub glitch and the diff didn’t update after the target branch was updated.